google/cloud/bigtable/table_admin_test.cc

```diff
@@ -14,9 +14,9 @@
 
 #include "google/cloud/bigtable/table_admin.h"
 #include "google/cloud/bigtable/grpc_error.h"
-#include "google/cloud/bigtable/internal/make_unique.h"
 #include "google/cloud/bigtable/testing/chrono_literals.h"
 #include "google/cloud/bigtable/testing/mock_admin_client.h"
+#include "google/cloud/internal/make_unique.h"
```

Comment:
It seems that `make_unique.h` file is included but never used, this inclusion can be avoided.

Reply:
Done.